File: comp/lvech/resultlvech.R
# Test script to verify lvech implementation matches MATLAB

# R implementation of lvech
lvech <- function(S) {
  return(S[lower.tri(S, diag = FALSE)])
}

# Create a simple test matrix (same structure in both languages)
test_matrix <- matrix(1:16, nrow=4)
print("Test matrix:")
print(test_matrix)

# Apply lvech function in R
r_result <- lvech(test_matrix)
print("R lvech result:")
print(r_result)

File: comp/lvech/resultlvech.m
% Save this as lvech_test.m in the same directory
% Test script to verify lvech implementation between MATLAB and R

% Create the same test matrix as in R
test_matrix = reshape(1:16, 4, 4);
disp('Test matrix:');
disp(test_matrix);

% Apply lvech function
matlab_result = lvech(test_matrix);
disp('MATLAB lvech result:');
disp(matlab_result);